### PR TITLE
Add LoongArch support for libpod

### PR DESCRIPTION
**Pull request by loongson-zjl**
Add Loongarch support for libpod.

[no new tests needed]

### CODE REVIEW

**Comment by rhatdan** [comment]
You need to add [no new tests needed]

**Comment by loongson-zjl** [comment]
> You need to add [no new tests needed]

how can I add this ?

**Comment by siretart** [comment]
You probably need amend your git commit to update the commit message and then git "force-push" your existing branch to replace your existing commit.

File: pkg/machine/fcos.go [hunk 1]
```diff
@@ -88,6 +88,8 @@ func GetFcosArch() string {
 	switch runtime.GOARCH {
 	case "arm64":
 		arch = "aarch64"
+	case "loong64":
+		arch = "loongarch64"
```
[the remark this answers is not in the document]
Comment (loongson-zjl) [reply pkg/machine/fcos.go:92]: thanks

**Comment by baude** [comment]
please rebase and sign your commit `git commit --amend -s` and force push

**Comment by loongson-zjl** [comment]
Ok,thank you . 


-----原始邮件-----
发件人:"Brent Baude" ***@***.***>
发送时间:2024-02-25 01:16:03 (星期日)
收件人: containers/podman ***@***.***>
抄送: "JiaLing Zhang" ***@***.***>, Author ***@***.***>
主题: Re: [containers/podman] Add LoongArch support for libpod (PR #21481)






please rebase and sign your commit git commit --amend -s and force push

**Comment by mheon** [comment]
Can you rebase? Should fix the tests

**Comment by baude** [comment]
@loongson-zjl rebase and we should be able to merge

**Comment by rhatdan** [comment]
I think @loongson-zjl has disappeared.

**Comment by loongson-zjl** [comment]
> git commit --amend -s

ok , Thank you.

**Comment by loongson-zjl** [comment]
> I think @loongson-zjl has disappeared.

ok, I'll do , thank you.

**Pull request merged**